versions.props

```diff
@@ -16,4 +16,5 @@ com.google.errorprone:error_prone_refaster = 2.3.3
 
 # Don't automatically force everything to upgrade slf4j
 org.slf4j:slf4j-api = 1.7.31
+org.apache.logging.log4j:* = 2.17.2
```

Comment:
Are we intentionally pinning on 2.17.2 for older support? Should we add a comment about when we'd want to unpin? [2.18.0](https://github.com/apache/logging-log4j2/blob/release-2.x/RELEASE-NOTES.md) has been out for about a month now and making its way through automated upgrades.

Reply:
Initially I hadn't used a compileOnly dependency, the goal here was to avoid pushing framework upgrades faster than our standard machinery might intend (which don't pin back releases, in fact the new version was bundled into products within hours of the release)

Reply:
Updated using the latest release, un-pinned